features/cisSysctl/file.include/etc/sysctl.d/99-cis.conf

```diff
@@ -11,7 +11,8 @@ net.ipv4.conf.all.rp_filter=1
 net.ipv4.conf.default.rp_filter=1
 net.ipv6.conf.all.disable_ipv6=1
 net.ipv6.conf.default.disable_ipv6=1
-net.ipv6.conf.lo.disable_ipv6=1
+net.ipv6.conf.lo.disable_ipv6=0
```

Comment:
why are we doing this? does not look like a setting we should disable.

Reply:
oh, the double negative. read it the wrong way. makes sense, thanks for clarifying